labs/instana/opentelemetry/5-application-monitoring/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 5. Application monitoring
+description: Configure Instana to monitor an Application
+sidebar_position: 6
+---
+
+# Application Perspective in Instana
+
+In the section of the labs, we will create an Application Perspective in Instana, so that we can see all of the metrics and traces
+collected from the OpenTelemetry Demo Application.
+
+An Application Perspective in Instana is a feature designed for monitoring, alerting, and analyzing microservice 
+environments. It allows users to dynamically scope the visibility of their applications to focus on specific services, 
+dependencies, technologies, business transactions, user journeys, deployment engines, versions, or releases.
+
+Key features and benefits of Application Perspectives:
+
+1. **Focused Monitoring**: Application Perspectives help teams stay focused on the services they are interested in, 
+reducing distractions from unrelated components. This targeted approach ensures that alerts, 
+errors, and logs are scoped to the relevant application perspective, streamlining troubleshooting.
+2. **Dynamic Scoping**: Users can specify subsets of services and their dependencies based on various criteria 
+such as zones, clusters, technologies, or cluster namespaces. This flexibility allows for customized monitoring setups 
+that meet specific needs.
+3. **Security and Visibility**: Organizations can use Application Perspectives to limit visibility to certain 
+infrastructure and services, enhancing security by controlling access to sensitive data.
+4. **Automated Dashboards**: Each Application Perspective auto-generates a feature-rich monitoring dashboard 
+that displays key metrics and golden signals, providing a comprehensive view of the application's health and performance.
+
+## Create an Application in Instana
+
+Let's create an application perspective in Instana. Log in to your Instana UI and create a new application perspective
+as described in the below picture
+
+![otel-015](images/otel-app-015.png) 
```

Comment:
Where there are a number of arrows and click sequence matters, for consistency and clarity, having the arrows numbered would be helpful to the student. See Ryan's examples for marking out click sequences: http://localhost:3000/waiops-tech-jam/labs/instana/server-and-agent-install-lab/server-installation/#34-preparing-the-server